inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -55,6 +57,10 @@ endif()
 set (PYTHON_BRIDGE_SRC_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 add_subdirectory (src/openvino/inference_engine)
 
+if(ENABLE_WHEEL)
+    add_subdirectory(install/wheel)
+endif()
```

Comment:
If I want to build wheels for multiple Python versions and rerun CMake specifying  `PYTHON_EXECUTABLE`
`PYTHON_INCLUDE_DIR`, `PYTHON_LIBRARY`, will I have multiple wheels?

Reply:
Yes, you will, but in this case, we would need to perform cmake install and setup.py several times

Reply:
This is fine tto run CMake again. I just mean if we need the same binaries but different Python version, it's OK not to clean buold folder for faster build. Just wanted to check if CMake is OK and nothing will stuck with wrong Python versions.